test-data/integration-tests/test_tfswitchtoml/.tfswitch.toml

```diff
@@ -1,3 +1,4 @@
 bin = "/usr/local/bin/terraform_from_toml"
-version = "0.11.4"
+version = "1.6.2"
 log-level = "NOTICE"
+product = "opentofu"
```

Comment:
Very much nitpick, but still: please add correct EOF
![image](https://github.com/warrensbox/terraform-switcher/assets/1274789/00fff1f8-8bf3-4a1f-9e70-b48a62282f0e)

Reply:
Fixed in 2a3ab61